<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-09-20 15:42:09.831 ***</b>
=ERROR REPORT==== 20-Sep-2024::15:42:09.831632 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.244062.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1973}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1934}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-09-20 15:42:09.832 ***</b>
=CRASH REPORT==== 20-Sep-2024::15:42:09.832267 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.244066.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2426)
    ancestors: [<0.244062.0>]
    message_queue_len: 0
    messages: []
    links: [<0.244062.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 6882
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-09-20 15:42:10.648 ***</b>
=WARNING REPORT==== 20-Sep-2024::15:42:10.648031 ===
'global' at node test_server@d21a2881ce2b requested disconnect from node 'gen_server_SUITE-multicall_remote-473250-11740@d21a2881ce2b' in order to prevent overlapping partitions

</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-09-20 15:42:47.944 ***</b>
=ERROR REPORT==== 20-Sep-2024::15:42:47.944145 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.248413.0&gt;,
                               {#Ref&lt;0.3485296778.2797862913.196722&gt;,1006893,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.3485296778.2797862913.196722&gt;,1006893,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-09-20 15:42:47.944 ***</b>
=CRASH REPORT==== 20-Sep-2024::15:42:47.944672 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.248434.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.3485296778.2797862913.196722&gt;,1006893,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [&lt;0.248413.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 4751
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2024-09-20 15:47:23.618 ***</b>
=CRASH REPORT==== 20-Sep-2024::15:47:23.618155 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.254198.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.3485296778.2799435777.125066&gt;,5322,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1694)
    ancestors: [&lt;0.254196.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 28118
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-09-20 15:47:32.734 ***</b>
=ERROR REPORT==== 20-Sep-2024::15:47:32.734289 ===
** Generic server <0.255477.0> terminating 
** Last message in was {nodedown,'peer_down_crash-441379-11740@d21a2881ce2b'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-441379-11740",
                              peer_down => crash},
                            'peer_down_crash-441379-11740@d21a2881ce2b',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,49,47,98,105,110,
                             47,101,114,108],
                            ["-sname","peer_down_crash-441379-11740",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQGQyMWEyODgxY2UyYgAD5fUAAAAAZu15Yw=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-441379-11740@d21a2881ce2b'}},
                            {<0.255463.0>,
                             #Ref<0.3485296778.2792357890.199950>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-441379-11740@d21a2881ce2b'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2024-09-20 15:47:32.734 ***</b>
=CRASH REPORT==== 20-Sep-2024::15:47:32.734788 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255477.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-441379-11740@d21a2881ce2b'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [<0.255463.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11217
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-09-20 15:47:33.660 ***</b>
=WARNING REPORT==== 20-Sep-2024::15:47:32.659063 ===
Process <0.91.0> on node 'peer_SUITE-shutdown_stop-441507-11740@d21a2881ce2b' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-09-20 15:47:34.017 ***</b>
=WARNING REPORT==== 20-Sep-2024::15:47:34.017579 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-441443-11740@d21a2881ce2b'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-09-20 15:47:34.176 ***</b>
=WARNING REPORT==== 20-Sep-2024::15:47:34.176642 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-426564-11740@d21a2881ce2b'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-09-20 15:47:50.337 ***</b>
=WARNING REPORT==== 20-Sep-2024::15:47:49.335959 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-442083-11740@d21a2881ce2b' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-09-20 15:47:50.757 ***</b>
=WARNING REPORT==== 20-Sep-2024::15:47:50.757514 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-426692-11740@d21a2881ce2b'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-09-20 15:47:50.809 ***</b>
=WARNING REPORT==== 20-Sep-2024::15:47:50.809552 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-402501-11740@d21a2881ce2b'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-09-20 15:47:53.495 ***</b>
=ERROR REPORT==== 20-Sep-2024::15:47:53.495453 ===
** Generic server <0.255831.0> terminating 
** Last message in was {'EXIT',<0.255827.0>,
                               {#Ref<0.3485296778.2799435777.139692>,1499456,
                                ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-484546-11740",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-484546-11740@d21a2881ce2b',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,49,47,98,105,110,47,101,114,108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-484546-11740",
                                     "-user","peer"],
                                    #Port<0.3115>,undefined,<<>>,running,
                                    {<0.255827.0>,
                                     #Ref<0.3485296778.2799435777.139782>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.3485296778.2799435777.139692>,1499456,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-09-20 15:47:53.496 ***</b>
=CRASH REPORT==== 20-Sep-2024::15:47:53.496003 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255831.0>
    registered_name: []
    exception exit: {#Ref<0.3485296778.2799435777.139692>,1499456,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.255827.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3115>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 13790
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-09-20 15:47:54.454 ***</b>
=WARNING REPORT==== 20-Sep-2024::15:47:53.452740 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-442403-11740@d21a2881ce2b' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-09-20 15:47:54.609 ***</b>
=WARNING REPORT==== 20-Sep-2024::15:47:54.609562 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-402725-11740@d21a2881ce2b'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-09-20 15:47:54.870 ***</b>
=WARNING REPORT==== 20-Sep-2024::15:47:54.870514 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-442371-11740@d21a2881ce2b'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-09-20 15:47:54.908 ***</b>
=WARNING REPORT==== 20-Sep-2024::15:47:54.908529 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-442435-11740@d21a2881ce2b'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-09-20 15:47:55.985 ***</b>
=ERROR REPORT==== 20-Sep-2024::15:47:55.984805 ===
** Generic server <0.255869.0> terminating 
** Last message in was {'EXIT',<0.255821.0>,
                           {#Ref<0.3485296778.2792095748.182050>,3988991,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "AYKLZXAVAWVPXPPZMRZA","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@d21a2881ce2b.2024-09-20_15.32.18/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-484578-11740",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@d21a2881ce2b.2024-09-20_15.32.18/make_test_dir.stdlib_test.logs/run.2024-09-20_15.32.54/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-484578-11740",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-25.3.2.14/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-484578-11740@d21a2881ce2b',
                                    "/usr/local/lib/erlang-25.3.2.14/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-484578-11740",
                                     "-setcookie","AYKLZXAVAWVPXPPZMRZA",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@d21a2881ce2b.2024-09-20_15.32.18/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-484578-11740",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@d21a2881ce2b.2024-09-20_15.32.18/make_test_dir.stdlib_test.logs/run.2024-09-20_15.32.54/log_private/",
                                     "-user","peer"],
                                    #Port<0.3142>,undefined,<<>>,running,
                                    {<0.255821.0>,
                                     #Ref<0.3485296778.2792357890.205301>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.3485296778.2792095748.182050>,3988991,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-09-20 15:47:55.985 ***</b>
=CRASH REPORT==== 20-Sep-2024::15:47:55.985688 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255869.0>
    registered_name: []
    exception exit: {#Ref<0.3485296778.2792095748.182050>,3988991,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.255821.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3142>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 17731
    stack_size: 29
    reductions: 18968
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-09-20 15:58:51.202 ***</b>
=ERROR REPORT==== 20-Sep-2024::15:58:51.201838 ===
** Generic server <0.302568.0> terminating 
** Last message in was {'EXIT',<0.302566.0>,
                               {#Ref<0.3485296778.2808348674.205028>,351104,
                                [{child1,<0.302575.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.302568.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.302574.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459160],
                            0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.3485296778.2808348674.205028>,351104,
    [{child1,<0.302575.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-09-20 15:58:51.203 ***</b>
=ERROR REPORT==== 20-Sep-2024::15:58:51.201833 ===
** Generic server <0.302569.0> terminating 
** Last message in was {'EXIT',<0.302566.0>,
                               {#Ref<0.3485296778.2808348674.205028>,351104,
                                [{child1,<0.302575.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.302569.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.302575.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459160],
                            0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.3485296778.2808348674.205028>,351104,
    [{child1,<0.302575.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-09-20 15:58:51.203 ***</b>
=CRASH REPORT==== 20-Sep-2024::15:58:51.203555 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.302568.0>
    registered_name: []
    exception exit: {#Ref<0.3485296778.2808348674.205028>,351104,
                     [{child1,<0.302575.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.302566.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 14825
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-09-20 15:58:51.204 ***</b>
=CRASH REPORT==== 20-Sep-2024::15:58:51.203856 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.302569.0>
    registered_name: []
    exception exit: {#Ref<0.3485296778.2808348674.205028>,351104,
                     [{child1,<0.302575.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.302566.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 14853
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_bridge_SUITE:simple_global_supervisor/1 2024-09-20 15:59:34.705 ***</b>
=CRASH REPORT==== 20-Sep-2024::15:59:34.705014 ===
  crasher:
    initial call: supervisor_bridge:supervisor_bridge_SUITE/1
    pid: <0.402726.0>
    registered_name: []
    exception exit: killed
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [bridge9212,<0.402721.0>]
    message_queue_len: 0
    messages: []
    links: [<0.402727.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4113
  neighbours:
    neighbour:
      pid: <0.402727.0>
      registered_name: work_supervisor_bridge_SUITE_server
      initial_call: {supervisor_bridge_SUITE,internal_loop_init,1}
      current_function: {supervisor_bridge_SUITE,internal_loop,1}
      ancestors: []
      message_queue_len: 0
      links: [<0.402726.0>]
      trap_exit: false
      status: waiting
      heap_size: 233
      stack_size: 3
      reductions: 9
      current_stacktrace: [{supervisor_bridge_SUITE,internal_loop,1,
                      [{file,"supervisor_bridge_SUITE.erl"},{line,152}]}]

</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2024 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Fri Sep 20 2024 15:32:54<!--/date--><br />
</div>
</center>
</body>
</html>
